### PR TITLE
Release modules

### PR DESCRIPTION
**Pull request by pcm32**
This PR moves and makes slight changes on the different modules used for the release. This is used as a submodule in Atlas Prod (and most content comes from there).

### CODE REVIEW

**Comment by pcm32** [comment]
This was used for the pre-release successfully.

File: bin/retrieve-expressions.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+[ ! -z ${dbConnection+x} ] || ( echo "Env var dbConnection needs to be defined." && exit 1 )
+[ ! -z ${ATLAS_EXPS+x} ] || ( echo "Env var ATLAS_EXPS path to the experiment needs to be defined." && exit 1 )
+
+echo "refresh materialized view VW_DIFFANALYTICS_DUMP WITH DATA;" | psql $dbConnection
+echo -e "Gene ID\tGene Name\tOrganism\tExperiment\tComparison id\tp-value\tlog2foldchange\tt-statistic" > $ATLAS_EXPS/differential_analytics.tsv
+echo "select IDENTIFIER, NAME, ORGANISM, EXPERIMENT, CONTRASTID, PVAL, LOG2FOLD, TSTAT from VW_DIFFANALYTICS_DUMP" \
+  | psql $dbConnection | tail -n +3 | perl -p -e 's|^ +||g' \
+  | perl -p -e 's/ +\| +/\t/g' | grep -v '^$' >> $ATLAS_EXPS/differential_analytics.tsv
+# Remove the last line (e.g. 5147277 rows selected.)
+sed -i '$ d'  $ATLAS_EXPS/differential_analytics.tsv
```
Comment (suhaibMo) [inline bin/retrieve-expressions.sh:15]: As indicated in the comments, its not needed anymore and this table has renamed with prefix old_VW_DIFFANALYTICS_DUMP.

**Comment by suhaibMo** [comment]
@pcm32 is there a main script which calls all these sub scripts to execute ?

**Comment by pcm32** [comment]
They are being called from different jenkins jobs, see the Bulk Data Release tab in Jenkins.

**Comment by suhaibMo** [comment]
Great I see it and it bleeds blue except OT which I understand `allOf failed: allOf failed: Not in enum list:` due to schema difference. LGTM !

**Review by suhaibMo — APPROVED** [review]
Good to go !

**Pull request merged**